rust/actyx/ax-core/src/util/version.rs

```diff
@@ -48,15 +48,21 @@ pub struct NodeVersion {
     pub git_hash: String,
 }
 
-const GIT_HASH: &str = match option_env!("AX_GIT_HASH") {
+// The hash is provided by GitHub actions, for more information, see:
+// https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+const GIT_HASH: &str = match option_env!("GIT_HASH") {
     Some(hash) => hash,
-    None => "",
+    // This is for cargo installations and builds
+    None => "cargo",
 };
 
 lazy_static! {
     pub static ref VERSION: String = NodeVersion::get().to_string();
```

Comment:
I think `struct NodeVersion` can be removed and this can be made into `const VERSION` using `concat!()`.

Reply:
Most likely, yes